### PR TITLE
Update blob_servers lists on scaling events

### PR DESCRIPTION
**Pull request by obino**

### CODE REVIEW

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/4566/

File: AppController/djinn.rb [hunk 1]
```diff
@@ -5029,6 +5034,9 @@ def scale_up_instances(needed_appservers)
         @last_scaling_time = Time.now.to_i
         Djinn.log_info("Added the following nodes: #{roles_needed}.")
       }
+
+      # Make sure we have the proper list of blobservers configured.
+      update_blob_servers
```
Comment (cdonati) [inline AppController/djinn.rb]: I don't think we can assume that the blob server is up and running on the new instance at this point.

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/4580/

**Comment by obino** [comment]
OCD build https://ocd.appscale.com:8080/job/Daily%20Build/4608/

**Pull request merged**